<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_PopupDial_IndicationChoice</class>
 <widget class="QWidget" name="C_PopupDial_IndicationChoice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>1</number>
   </property>
   <item row="3" column="0" colspan="2">
    <widget class="QTreeWidget" name="treeWidget_Indications">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Indications liées au produit</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ASMR/SMR/Remb</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Commentaire</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Remb</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>cip</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="checkBox_IndicationBDM">
     <property name="whatsThis">
      <string>Evaluer les indications spécifiques à l'implémentation commerciale des données</string>
     </property>
     <property name="text">
      <string>Spécifique à la base</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBox_IndicationCIM10">
     <property name="whatsThis">
      <string>Evaluer les indications codées avec le CIM10</string>
     </property>
     <property name="text">
      <string>CIM 10</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBox_SMR">
     <property name="toolTip">
      <string>Evaluer le SMR (service médical rendu)</string>
     </property>
     <property name="whatsThis">
      <string>Evaluer le SMR (service médical rendu)</string>
     </property>
     <property name="text">
      <string>SMR</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="checkBox_ASMR">
     <property name="whatsThis">
      <string>Evaluer l'ASMR (l'ajout de service médical rendu)</string>
     </property>
     <property name="text">
      <string>ASMR</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTextEdit" name="textEdit_memo">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Informations sur l'indication</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
